integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ReceiveAlarmNotifications.feature

```diff
@@ -8,17 +8,28 @@ Feature: SmartMetering - Receive Alarm Notifications
     Given a dlms device
       | DeviceIdentification | TEST1024000000001 |
       | DeviceType           | SMART_METER_E     |
+    And a dlms device
+      | DeviceIdentification | TEST102600000001  |
+      | DeviceType           | SMART_METER_E     |
 
   Scenario: Handle a received alarm notification from a known device
     When an alarm notification is received from a known device
       | DeviceIdentification | TEST1024000000001 |
-    Then the alarm should be pushed to the osgp_logging database device_log_item table
+    Then 2 alarm should be pushed to the osgp_logging database device_log_item table
```

Comment:
Why expect 2 alarms here and only one for mx382?

Reply:
This is not the test for mx382 alarm notifications. This is an existing test for an other type of notification in which a preceeding test step creates 2 notifications. I rewrote this test step with a parameterized  number of alarms so it could be used for testing mx382 alarm notifications as well.